apps/svelte.dev/content/tutorial/01-svelte/04-logic/05-keyed-each-blocks/index.md

```diff
@@ -13,7 +13,7 @@ Click the 'Remove first thing' button a few times, and notice what happens:
 
 > [!NOTE] If you're coming from React, this might seem strange, because you're used to the entire component re-rendering when state changes. Svelte works differently: the component 'runs' once, and subsequent updates are 'fine-grained'. This makes things faster and gives you more control.
 
-One way to fix it would be to make `emoji` a [`$derived`](derived-state) value. But it makes more sense to remove the first `<Thing>` component altogether than to remove the _last_ one and update all the others.
+One way to fix it would be to make `emoji` a [`$derived`](derived-state) value. But it makes more sense to remove the first `<Thing>` component altogether rather than to remove the _last_ one and update all the others.
```

Comment:
can't have both 'rather' and 'to', have to choose

```suggestion
One way to fix it would be to make `emoji` a [`$derived`](derived-state) value. But it makes more sense to remove the first `<Thing>` component altogether rather than remove the _last_ one and update all the others.
```